root/io/filemerger/execTestMultiMerge.C

```diff
@@ -67,26 +67,27 @@ int testSimpleFile(const char *filename, Long64_t entries, Int_t compSetting, Lo
    return 0;
 }
 
-
 int execTestMultiMerge()
 {
    Int_t result = 0;
-   result += testMergedFile("mfile1-4.root",1,4938, kIs32bits ? 3 : 0);
-   result += testMergedFile("mzfile1-4.root",206,4992, kIs32bits ? 2 : 0);
-
-   result += testSimpleFile("hsimple.root",25000,1,414415, kIs32bits ? 10 : 8);
-   result += testSimpleFile("hsimple9.root",25000,9,432029,3);
-   result += testSimpleFile("hsimple9x2.root",2*25000,9,851123,9);
-   result += testSimpleFile("hsimple209.root",25000,209,394077,8);
-   result += testSimpleFile("hsimpleK.root",5*25000,209,1917395,16);
+   int hsimpleFTolerance = 6; // 5 for non fst builds
+   //result += testMergedFile("mfile1-4.root",1,4938, kIs32bits ? 3 : 0);
+   result += testMergedFile("mzfile1-4.root",206,5819, kIs32bits ? 2 : 0);
+   result += testMergedFile("mlz4file1-4.root",406,5840, kIs32bits ? 2 : 0);
+   result += testMergedFile("mzlibfile1-4.root",106,5744, kIs32bits ? 2 : 0);
+   result += testSimpleFile("hsimple.root",25000,1,519464, kIs32bits ? 10 : 8);
+   result += testSimpleFile("hsimple9.root",25000,9,518137,3);
+   result += testSimpleFile("hsimple9x2.root",2*25000,9,1020462,9);
+   result += testSimpleFile("hsimple209.root",25000,209,395932,8);
+   result += testSimpleFile("hsimple409.root",25000,409,518144,8);
+   result += testSimpleFile("hsimpleK.root",6*25000,209,2300831,16);
    if (lzma_version_number() < 50020010) {
       // lzma v5.2.0 produced larger files ...
       // but even older version (eg v5.0.0) produced smaller files ...
-      result += testSimpleFile("hsimpleK202.root",5*25000,202,1938720,700);
+      result += testSimpleFile("hsimpleK202.root",6*25000,202,1938720,700);
```

Comment:
I hate to ask and apologize for the piecemeal requests.  Can we add
hsimple101
hsimple106
hsimple401
hsimple406

Thanks :)